### ISSUE TITLE
Interactive console for Limnoria

### ISSUE THREAD

**Issue by techtonik** [progval/Limnoria#1252]
It would be nice to have command line REPL for testing purposes.
Expected:

```
$ python -m limnoria
Limnoria Bot v2016.06.29
>>> hello
olleh
```

Actual:

```
$ python -m limnoria
python: No module named limnoria
```

**Comment by GLolol** [comment]
Can you clarify what you mean by "REPL"?

**Comment by techtonik** [comment]
Sorry. The term was not clear for me either for the first time. It is translated as "Read–eval–print loop", and in layman terms it is a simple interactive shell/console - https://pymotw.com/2/cmd/

**Comment by ProgVal** [comment]
What would this REPL do? Read commands and answer them?

**Comment by techtonik** [comment]
@ProgVal yes, for quick testing and teaching bot. Maybe for tests.

**Comment by GLolol** [comment]
I'm assuming you mean regular bot commands. But what does "teaching bot" entail?

**Comment by techtonik** [comment]
I can't explore Limnoria features, but some plugins/bots like [infobot](http://www.infobot.org/guide-0.43.x.html) allow to set facts interactively. So REPL is just a way to implement chat session with the bot from console. i.e. without IRC.
